crates/liboci-cli/src/checkpoint.rs

```diff
@@ -27,4 +28,28 @@ pub struct Checkpoint {
     /// Path for saving work files and logs
     #[clap(long)]
     pub work_path: Option<PathBuf>,
+    /// Path for previous criu image file in pre-dump
+    #[clap(long)]
+    pub parent_path: Option<PathBuf>,
+    /// Use lazy migration mechanism
+    #[clap(long)]
+    pub lazy_pages: bool,
+    /// Pass a file descriptor fd to criu
+    #[clap(long)]
+    pub status_fd: Option<u32>, // TODO: Is u32 the right type?
```

Comment:
For what it's worth, I am not sure how file descriptors should be passed around in that context.